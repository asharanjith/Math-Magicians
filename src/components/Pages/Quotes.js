import React from 'react';

const Quotes = () => (
  <div>
    <p className="Quotes">
      Mathematics is not about numbers, equations ,computations, or algorithms
      ,it is about understanding - William Paul Thurstin.
    </p>
  </div>
);

export default Quotes;
