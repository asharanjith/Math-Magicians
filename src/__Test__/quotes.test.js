import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../components/Pages/Quotes';

describe('check Quotes', () => {
  const tree = renderer.create(<Quotes />).toJSON();
  test('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
