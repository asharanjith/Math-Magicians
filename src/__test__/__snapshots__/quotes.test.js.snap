// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check Quotes renders correctly 1`] = `
<div>
  <p
    className="Quotes"
  >
    Mathematics is not about numbers, equations ,computations, or algorithms ,it is about understanding - William Paul Thurstin.
  </p>
</div>
`;
